src/base/bootstrap-overrides/_base.scss

```diff
@@ -831,3 +831,10 @@ blockquote {
 		}
 	}
 }
+
+caption {
+	text-align: center;
+	color: #333;
+	font-weight: bold;
+	font-size: 1.1em;
```

Comment:
this is failing because of SCSS lint please rewrite to (needs to be alphabetically ordered

```scss
color: #333;
font: {
	size: 1.1em;
	weight: bold;
}
text-align: center;
```